README.md

```diff
@@ -857,6 +857,7 @@ Below is the complete list of available options that can be used to customize yo
 | `GITLAB_BACKUP_ARCHIVE_PERMISSIONS` | Sets the permissions of the backup archives. Defaults to `0600`. [See](http://doc.gitlab.com/ce/raketasks/backup_restore.html#backup-archive-permissions) |
 | `GITLAB_BACKUP_TIME` | Set a time for the automatic backups in `HH:MM` format. Defaults to `04:00`. |
 | `GITLAB_BACKUP_SKIP` | Specified sections are skipped by the backups. Defaults to empty, i.e. `lfs,uploads`. [See](http://doc.gitlab.com/ce/raketasks/backup_restore.html#create-a-backup-of-the-gitlab-system) |
+| `GITLAB_BACKUP_STRATEGY` | Set to `copy` to avoid `file changed as we read it` errors. [See](https://docs.gitlab.com/ee/raketasks/backup_restore.html#backup-strategy-option) |
```

Comment:
Here should be the link to the  documentation for the community edition and not enterprise edition replace this with this `https://docs.gitlab.com/ce/raketasks/backup_restore.html#backup-strategy-option`